### PR TITLE
Add production config to gitlab ci file

### PR DESCRIPTION
**Pull request by tebbon**
## Description :sparkles:
Add the production build and deployment configuration to the gitlab CI script.

### Related :handshake:
**[VEN-868](https://helsinkisolutionoffice.atlassian.net/browse/VEN-868): Execute the new infra switchover**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -82,3 +82,30 @@ staging:
     K8S_SECRET_ELASTIC_APM_SECRET_TOKEN: "$GL_QA_ELASTIC_APM_TOKEN"
     K8S_SECRET_ELASTIC_APM_SERVER_URL: "$GL_QA_ELASTIC_APM_URL"
     K8S_SECRET_ELASTIC_APM_SERVICE_NAME: "berths-backend"
+
```
Comment (santtul) [inline .gitlab-ci.yml:85]: Could we remove the `test-codeanalysis:` part above?

Comment (tebbon) [reply .gitlab-ci.yml:85]: yes, if we don't like it...

Comment (santtul) [reply .gitlab-ci.yml:85]: Let's remove it because it's already run by Travis, no need to run it again in this stage.

**Review by santtul — APPROVED** [review]
🤖 Approved by LGTM bot

**Pull request merged**